packages/visualizations/src/components/Table/utils.ts

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { get } from 'svelte/store';
+import { isNil } from 'lodash';
+import type { DataFormat, Column, ColumnOfType } from './types';
+import { lastStickyColumn } from './store';
+
+export const getStickyClasses = (column: Column, scrolled: boolean) =>
```

Comment:
I really don't get how this could happen, but once loaded in the platform, we get the following class attribute on `th`:
```
table-header-- undefined false false svelte-1quk48x
```
How this function could return `undefined false false` eludes me 😳

Reply:
really seems like it was not on the last version (I just checked it again) 🤔

Reply:
Weird, I'll try again and inspect my node_modules. There's so many layers of indirection 🙈